compiler/src/iree/compiler/Codegen/Sandbox/LinalgTensorCodegenDriver.cpp

```diff
@@ -37,16 +37,25 @@ using mlir::iree_compiler::IREE::LinalgExt::CodegenStrategy;
 //===----------------------------------------------------------------------===//
 
 /// Returns the op that contains lowering config. Returns failure if there are
-/// multiple op having lowering config.
+/// multiple op having a different lowering config. Otherwise, return the first
+/// root op we found.
 static FailureOr<Operation *> getRootOp(func::FuncOp funcOp) {
   Operation *rootOp = nullptr;
   auto result = funcOp.walk([&](Operation *op) -> WalkResult {
     if (!iree_compiler::getLoweringConfig(op)) return WalkResult::advance();
     if (rootOp) {
-      return WalkResult::interrupt();
```

Comment:
Lets start using this on the CUDA side since that is the need. We can drop the use here since there is no plan to use this on the CPU side for now.